config/zones/NC.yaml

```diff
@@ -1,3 +1,6 @@
 country: NC
+disclaimer: 'The production reported for this zone does not include self-production and consumption by metallurgy plants in New Caledonia.'
```

Comment:
Do we have a rough idea of how much this accounts for? Is it a negligible amount? We usually don't care about production/consumption that isn't connected to the grid

Reply:
It's not, it's was 75% a few years ago. Should I add the percentages to the disclaimer as well?

Reply:
Oh okay, then we should probably keep the disclaimer. Are the plants electric? Are we sure that it isn't direct emissions (i.e. accounted for in energy instead of electricity)?

Reply:
I don't think the plants are electric but rather that the production consumption on the rest of the island is pretty low in comparison. So even if it's not fully electric all of the heavy machinery and processes consume most of it. But I'm sure some are used for heat as well. 

From the original issue:
> Major caveat: doesn't include electricity consumption of mines and metallurgy, which [this 2018 Enercal report (page 17)](https://www.enercal.nc/sites/portail/files/atoms/files/enercal_-_bilan_2018_et_perspectives_du_systeme_electrique_nc.pdf) says is 76% of total (with public distribution the other 24%); it's now 75%.[[2]](https://www.enercal.nc/la-transition-energetique-en-nouvelle-caledonie/mix-energetique-emissions-de-co2/)

I'll update the disclaimer to mention mines as well, I realise I forgot that before.

Reply:
Probably worth noting "which ensures the stability of the grid" implies that these plants are, in fact, grid-connected, but from what I understood the mines & metallurgy sector consumes the  same amount of electricity it produces.